packages/generator/src/action-function-import/parameters-interface.ts

```diff
@@ -1,25 +1,25 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
 import { InterfaceDeclarationStructure, StructureKind } from 'ts-morph';
-import { VdmFunctionImport } from '../vdm-types';
+import { VdmActionImport, VdmFunctionImport } from '../vdm-types';
 import { addLeadingNewline } from '../typedoc';
 
-export function functionImportParametersInterface(
-  functionImport: VdmFunctionImport
+export function parametersInterface(
+  actionFunctionImport: VdmFunctionImport | VdmActionImport
 ): InterfaceDeclarationStructure {
   return {
     kind: StructureKind.Interface,
-    name: functionImport.parametersTypeName,
+    name: actionFunctionImport.parametersTypeName,
     isExported: true,
-    properties: functionImport.parameters.map(parameter => ({
+    properties: actionFunctionImport.parameters.map(parameter => ({
       name: parameter.parameterName,
       type: parameter.jsType,
       hasQuestionToken: parameter.nullable,
       docs: [addLeadingNewline(parameter.description)]
     })),
     docs: [
       addLeadingNewline(
-        `Type of the parameters to be passed to [[${functionImport.functionName}]].`
+        `Type of the parameters to be passed to [[${actionFunctionImport.name}]].`
```

Comment:
Btw - one more reason to call it parameters instead of payload.

Reply:
yes